process/block/baseProcess.go

```diff
@@ -211,7 +211,7 @@ func (bp *baseProcessor) isHdrConstructionValid(currHdr, prevHdr data.HeaderHand
 	}
 
 	if !bytes.Equal(currHdr.GetPrevHash(), prevHeaderHash) {
-		return process.ErrInvalidBlockHash
+		return process.ErrInvalidNotarizedBlockHash
```

Comment:
ErrInvalidNotarizedBlockHash? Is it invalid, or they do not match?

Reply:
Actually it could be ErrBlockHashDoesNotMatch

Reply:
Renamed

Reply:
ok